services/explorer/src/api/graphql/queries/dagBlocks.ts

```diff
@@ -1,15 +1,54 @@
 export const dagBlockQuery = `
-  query dag_block_query($hash: Bytes32) {
-    dagBlock (hash: $hash) {
-      hash,
-      pivot,
-      tips,
-      level,
-      pbftPeriod,
-      timestamp,
-      transactionCount,
+query dag_block_query($hash: Bytes32) {
+  dagBlock (hash: "0c9d36dd14c50e0c319e095f06698a05bb80def51c7927d783514b051c619454") {

```

Comment:
You probabbly forgot this when testing.
We need to have the variable $hash here